src/TextareaAutosize.js

```diff
@@ -3,9 +3,11 @@
  */
 
 import React from 'react';
-import emptyFunction from 'react/lib/emptyFunction';
+import ReactDOM from 'react-dom';
```

Comment:
Thanks @jquense for the insights. So how should we handle this? I guess the only reasonable way to go is to wait until react and react-dom 0.14 is stable and then merge the PR and remove the support for 0.13. Any other ideas?